webpack.config.ts

```diff
@@ -293,6 +293,16 @@ const appConfig: Configuration = {
         test: /\.(woff|woff2|ttf|eot|svg|png|gif|ico|jpg|mp4)($|\?)/,
         type: 'asset',
       },
+      {
+        test: /\.mdx?$/,
+        use: [
+          {
+            loader: '@mdx-js/loader',
+            /** @type {import('@mdx-js/loader').Options} */
```

Comment:
we're using typescript can import the type probably, but we're not using any options